package.json

```diff
@@ -10,7 +10,8 @@
     "start:js": "webpack --progress --watch",
     "start": "npm run start:js & bundle exec jekyll serve --livereload",
     "test:a11y-desktop": "pa11y-ci --config ./.pa11yci-desktop --sitemap http://127.0.0.1:4000/sitemap.xml",
-    "test:a11y-mobile": "pa11y-ci --config ./.pa11yci-mobile --sitemap http://127.0.0.1:4000/sitemap.xml"
+    "test:a11y-mobile": "pa11y-ci --config ./.pa11yci-mobile --sitemap http://127.0.0.1:4000/sitemap.xml",
```

Comment:
@eileen-nava Right, so what's happening here is that to do the pa11y testing locally you need to have an instance of the dev server running. It will run the tests but will probably fail because it can't find the files. When this happens it seems to be appending an extra `/` to the beginning of the file path. I'm not sure what's going on here. This could be a problem worth investigating in another ticket. Thanks for flagging!